pkg/backend/display/progress.go

```diff
@@ -964,10 +964,12 @@ func (display *ProgressDisplay) processNormalEvent(event engine.Event) {
 
 		// Register the resource update start time to calculate duration
 		// and time elapsed.
+		display.m.Lock()
```

Comment:
Should we use a separate lock for this?  This lock seems to be used only for `eventUrnToResourceRow` according to the docs. I'm guessing this is convenient because it already does the read locking for `opStopwatch`, and that's why we don't need to add it in this PR?

It's probably okay to reuse the lock in that case, but we should update the comment.

Reply:
@PollRobots I _believe_ you have been working with display code recently and have a lot more experience than I do here -- any thoughts?

Reply:
I think just update the comment and it'll be fine.